desk/gembind.c

```diff
@@ -190,21 +72,34 @@ static LONG             addr_out[AO_SIZE];
 
 
 
-
-
-static WORD gem_if(WORD opcode)
+static __inline__ void gem(void)
 {
-    WORD i;
-    const BYTE *pctrl;
+	register long pb __asm__("d1") = (long)&gb;
+    __asm__ volatile (
+        "move.w  %1,d0\n\t"
+        "trap    #2\n\t"
+         :
+         : "r"(pb), "i"(200)
+         : "d0", "d2", "a0", "a1", "a2", "memory", "cc"
```

Comment:
I saw that in MiNTLib's sources: GCC 2.95 required (or just allowed?) to put output registers in clobber list. Someone introduced that __CLOBBER_RETURN() macro. So I also used it. Now I wonder if adding output registers to clobber list really matters with GCC 2.95, this should be tested again.